### DISCUSSION TITLE
Is this project abandoned / dead?

### DISCUSSION THREAD

**Discussion by alekslyse** [jesec/flood#644] [General]
In a lack of a good torrent UI, I see this one had not been updated since 2001. Is it abandoned?

**Comment by awsms** [comment]
Yes, jesec has not been active for months.

**Comment by artecnico** [comment]
Past Sunday 11 he commited some changes to the repository. Maybe he is back?

**Reply by trim21** [reply]
he is not back, but I just got permission to merge pr and push tags.

**Comment by newadventure079** [comment]
Wow, 2001 is a long time to be inactive!
